extensions/wikia/AdEngine/js/lookup/prebid/adapters/aol.js

```diff
@@ -9,6 +9,7 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.aol', [
 
 	var bidderName = 'aol',
 		network = '9435.1',
+		priorityLevel = 1,
```

Comment:
what do you think about returning `1` by default in all adapters?

Reply:
We can do that. Reverted and refactored.